compareECLFiles.cmake

```diff
@@ -378,6 +378,13 @@ add_test_compareECLFiles(CASENAME editnnc_model2
 			  REL_TOL ${rel_tol}
 			  DIR model2)
 
+add_test_compareECLFiles(CASENAME wsegsicd
+			  FILENAME TEST_WSEGSICD.DATA
+			  SIMULATOR flow
+			  ABS_TOL ${abs_tol}
+			  REL_TOL ${rel_tol}
+			  DIR wsegsicd)
```

Comment:
not necessary to specify DIR if it matches the casename.

Reply:
okay. I pushed it to the wrong branch even. Thanks for commenting.